deployments/kubectl/configMap.yaml

```diff
@@ -112,3 +112,286 @@ data:
                 "type": "object"
             }
         }
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: openfga-schema
+data:
+  schema.json: |
```

Comment:
I don't like this, we would have to update this json every time we make a change to the schema. Can't we use the CLI: `identity-platform-admin-ui create-fga-model` for this? (we could check if the `OPENFGA_AUTHORIZATION_MODEL_ID` file exists and if not we run this command)